### PR TITLE
add "space parent" picker

### PR DESCRIPTION
**Pull request by guscost**

### CODE REVIEW

**Comment by 1egoman** [comment]
Rob says that he's prefer this to be a select box, but it's likely we will merge this anyway and then will fix it up at a later date.

**Comment by guscost** [comment]
There are mockups?

**Comment by guscost** [comment]
Aha, so there are

File: src/components/admin-locations-detail-modules/styles.module.scss [hunk 1]
```diff
@@ -8,6 +8,8 @@
   width: 818px;
   border: 1px solid $grayLight;
   border-radius: $borderRadiusBase;
+
+  // overflow: hidden;
```
Comment (1egoman) [inline src/components/admin-locations-detail-modules/styles.module.scss]: Can this be removed?

**Pull request merged**